KAD-4494 Add justify-content support for vertical alignment in colum…

gilbert-hernandez opened the pull request:
[https://stellarwp.atlassian.net/browse/KAD-4494](https://stellarwp.atlassian.net/browse/KAD-4494)

added checks for justify content for vertical alignment for tablet and mobile devices.

mark-c-woodard reviewed (changes requested):
https://www.loom.com/share/d8bc3e0a3681495eb9068a79a9f15b2e?sid=fceb16fc-4aa6-44dc-9e33-70a0056dc2c5

gilbert-hernandez commented:
> https://www.loom.com/share/d8bc3e0a3681495eb9068a79a9f15b2e?sid=fceb16fc-4aa6-44dc-9e33-70a0056dc2c5

@mark-c-woodard 
I committed another change for this issue you brought up. I removed a check for vertical alignment for tablet which seems to work and updated the justify content for mobile when horizontal. This played into the issue when tablet wasn't set but desktop and mobile were.

gilbert-hernandez commented on includes/blocks/class-kadence-blocks-column-block.php:
```diff
@@ -818,6 +816,7 @@ public function build_css( $attributes, $css, $unique_id, $unique_style_id ) {
 			$css->set_selector( '.kadence-column' . $unique_id . ' > .kt-inside-inner-col' );
 			$css->add_property( 'flex-direction', ( 'horizontal-reverse' === $mobile_direction ? 'row-reverse' : 'row' ) );
 			$css->add_property( 'flex-wrap', 'wrap' );
+			$css->add_property( 'justify-content', 'flex-start' );
```
This was added for a similar situation as in the video above. When the desktop direction is set to vertical, the tablet direction is left unset, and the mobile direction is set horizontal, the justification is centered by default. In the editor, the text is aligned to the left or right depending on if it is horizontal or horizontal-reversed. So, this sets a default flex-start justification which is changed later in the code with a switch that checks $mobile_text_align.
